OpenUpSA/wazimap-ng
# cms app for tutorial text configuration #189

## Description
New CMS app to configure user pages.
Will be used for tutorial for now but it should be dynamic enough to let user create content for available pages and content is sent through the api.

Modal Structure of app:

  * Page (only visible to SuperAdmin)
         -- Name
         --  FK to profile
         -- api_mapping (where to put the content)

 * Content
         -- Title
         --  FK to page
         --  Image
         --  Text
         --  order (edited) 

Permissions:
* Pages can only be added by superadmin
* content will be added by profile admins
* Profile admin can only see/edit content for pages enabled for there profile

Api:
* Page model should have serializer.

## Related Issue
#189 

## How to test it locally
Admin panel permissions test to check if content is only available for profile admins of a profile:
* Create new page for a profile
* Create multiple content pages for a page
* create a profile admin without group for Page obj profile and check contents in cms app.
* It should be empty now add profile group to logged in user and user should be able to view content pages

Api:
* Create page & content for a profile
* api of profile should include page["api_mapping"] = page content obj list in configuration field of profile

## Changelog
* Added new app cms including admin view, serializer and models
* Changed profile serializer to include page in configuration


### Added

### Updated

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
